src/frontend/components/button/lib/common.ts

```diff
@@ -0,0 +1,32 @@
+import gadsStorage from "util/gadsStorage";
+
+export async function clearSavedFormValues($form: JQuery<HTMLElement>) {
```

Comment:
Just for my benefit, can you briefly comment what this function is used for and/or does?